pkgs/development/interpreters/dart/default.nix

```diff
@@ -1,87 +1,34 @@
-{ stdenv, fetchurl, unzip, version ? "2.7.2" }:
-
-let
-
-  sources = let
-
+{ stdenv
+, fetchurl
+, unzip
+, version ? "2.10.4"
```

Comment:
Not sure if we want to default to a version. Maybe we should find a different way.

Reply:
Why not? I think we want to support only the version of Dart compatible with Flutter.

Reply:
I am not sure how common it is to overwrite this but maybe we can remove it entirely. Not sure if it is worth it.

Reply:
I think it is worth it to keep it because it is quite common for the Dart is Nixpkgs to not match the Dart version you need it. It is just that almost everyone out there will need a specific version of Dart so we can't support just, like, the 2 or 3 most used versions.

Reply:
The idea is, we will support the most recent stable version. If you do need another version, fine, you can call this derivation with the correct parameters and it may work (or not).